### ISSUE TITLE
allow specifying any jvm arguments for starting JVM

### ISSUE THREAD

**Issue by behrica** [behrica/clojurebridge#4]
at least "java properties" but as well all arguments related to "java module system".

Workaround is to set environment variable "_JAVA_OPTIONS

```bash
export _JAVA_OPTIONS="-Dlibpython_clj.manual_gil=true"
```

**Comment by behrica** [comment]
One issue preventing to advance is this:

https://github.com/CellProfiler/python-javabridge/issues/19

even when PR https://github.com/clj-python/libpython-clj/pull/228

gets merged.

Certain forms of "aliases", like:
```
:jdk-17
           {:jvm-opts ["--add-modules" "jdk.incubator.foreign"]}
```
might not work.

while this will work:
```
:jdk-17
           {:jvm-opts ["--add-modules=jdk.incubator.foreign"]}
```

**Comment by behrica** [comment]
closed in 0.0.8
with commit 91510d69b5384350e73fa10da18fcecbb84fb545

The above issue on the "format" of the JVM properties is still open.
https://github.com/CellProfiler/python-javabridge/issues/19

Workaround is to specify the jvm properties in the alias as

```
{:jvm-opts ["--add-modules=jdk.incubator.foreign"]}
```

and **not** as
```
{:jvm-opts ["--add-modules" "jdk.incubator.foreign"]}
```
Clojure works with both in the same way.
